backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceAttributes.java

```diff
@@ -48,9 +48,15 @@ public BacktraceAttributes(Context context, BacktraceReport report, Map<String,
             clientAttributes) {
         this.context = context;
         if (report != null) {
-            this.convertAttributes(report, clientAttributes);
+            this.convertReportAttributes(report);
             this.setExceptionAttributes(report);
         }
+        if (clientAttributes != null) {
```

Comment:
Why do we need this change in this diff? What is the purpose of this change?

Reply:
The purpose of this change is that we don't need to have a valid `report` to take the `clientAttributes`. If we have valid `clientAttributes`, we should include those in the new `BacktraceAttributes` even if we don't have a `report`. 

This was preventing `BacktraceAttributes` from being constructed properly for Crashpad. Because we were calling `BacktraceAttributes` constructor with a null `report`, so the `clientAttributes` were not getting passed to Crashpad. Now with this change, we do get the `clientAttributes` from Crashpad, even though we call `BacktraceAttributes` constructor with a null `report`.

Reply:
oh now I understand! thanks for explanation Rameez!